pkg/local_workflows/code_workflow.go

```diff
@@ -60,7 +60,7 @@ func codeWorkflowEntryPoint(invocationCtx workflow.InvocationContext, _ []workfl
 	if ignoresFeatureFlag && !reportEnabled {
 		logger.Debug().Msg("Implementation: Native")
 
-		unsupportedParameter := []string{"severity-threshold", "project-name", "project-id", "commit-id", "target-name", "target-file", "remote-repo-url"}
+		unsupportedParameter := []string{"project-name", "project-id", "commit-id", "target-name", "target-file"}
```

Comment:
question: does this PR add support for `severity-threshold` too? Is it worth mentioning in a commit?

Reply:
No it doesn't, I stumbled across this leftover warning. The actual support for the argument is already merged.